pages/admin/admin-management.tsx

```diff
@@ -240,6 +265,7 @@ export default function AdminManagement() {
   const [recordsCount, setRecordsCount] = useState(0);
 
   const { refetch } = useQuery<GetEmployeesResponse, GetEmployeesRequest>(GET_EMPLOYEES_QUERY, {
+    fetchPolicy: 'cache-and-network',
```

Comment:
I'm wondering if there was a specific reason for using this fetch policy? I'm guessing it was to address the table not being updated upon refetching (following the mutation call). In our other queries, we have resolved this by setting `notifyOnNetworkStatusChange` to `true`. https://www.apollographql.com/docs/react/data/queries/#inspecting-loading-states

Reply:
Yup, that is correct - I added it because the table was not updating upon refetching. Not sure why it wasn't working on my end but I have removed it since it seems to work now? I think just putting notifyOnNetworkStatusChange makes sense!